src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/index.jsx

```diff
@@ -30,35 +31,59 @@ const initialSchema = {
 };
 
 const pageKey = 'ccPreferences';
-const pageTitle = 'Tell us your community care preferences';
+let pageTitle = 'Tell us your community care preferences';
 
 export default function CommunityCareProviderSelectionPage() {
   const dispatch = useDispatch();
   const { data, pageChangeInProgress, schema, showCCIterations } = useSelector(
     state => getFormPageInfo(state, pageKey),
     shallowEqual,
   );
+  const residentialAddress = useSelector(
+    state => selectVAPResidentialAddress(state),
+    shallowEqual,
+  );
   const history = useHistory();
 
+  if (showCCIterations && residentialAddress) {
+    // TODO: Should the type of care name be lower case???
+    pageTitle = `Request a ${getTypeOfCare(data).name} provider`;
+  }
   const descriptionText = showCCIterations
     ? 'We’ll call you to confirm your provider choice or to help you choose a provider if you skip this step.'
     : 'You can request a provider for this care. If they aren’t available, we’ll schedule your appointment with a provider close to your home.';
 
-  const uiSchema = {
-    communityCareSystemId: {
-      'ui:title': 'What’s the closest city and state to you?',
-      'ui:widget': 'radio',
-    },
-    communityCareProvider: {
-      'ui:options': {
-        showFieldLabel: true,
+  let uiSchema;
+  if (showCCIterations) {
```

Comment:
It looks like this change is to remove the city/state question, but I think that's handled elsewhere. Does removing this change cause any issues?

Reply:
Yes. removing the change displays this:

![Screen Shot 2021-09-19 at 7 50 03 PM](https://user-images.githubusercontent.com/3587066/133948619-0a3205cd-eeb8-4231-80c7-ea1b33b18031.png)

'Request a Primary care provider. (Optional)' should not be displayed.

Reply:
I added Jennie's changes.